src/models/visits.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model } = require('mongoose');
+
+const visitSchema = Schema({
+	clientId: {
+		type: Schema.Types.ObjectId,
+		ref: 'Client',

```

Comment:
Are u sure it'll work?
I think you need to write a name the same as it's in the database.

Reply:
need to discuss

Reply:
Need a fix to "client".